rest-api-spec/src/main/resources/rest-api-spec/test/search/330_fetch_fields.yml

```diff
@@ -0,0 +1,42 @@
+setup:
+  - do:
+      indices.create:
+          index:  test
+          body:
+            mappings:
+              properties:
+                keyword:
+                  type: keyword
+                integer_range:
+                  type: integer_range
+
+  - do:
+      index:
+          index:  test
+          id:     1
+          body:
+            keyword: [ "first", "second" ]
+            integer_range:
+              gte: 0
+              lte: 42
+
+  - do:
+      indices.refresh:
+          index: [ test ]
+
+---
+"Test basic field retrieval":
+  - do:
+      search:
+        index: test
+        body:
+          fields: [keyword, integer_range]
```

Comment:
We'll no longer use a flat list of fields once we introduce support for formatting the values.

Reply:
You just have this temporarily because it is simpler to parse?